services/app/src/static/style/gameScore.css

```diff
@@ -0,0 +1,9 @@
+.game-score {
+  position: relative;
```

Comment:
Not an expert on css positioning, but `relative` position will probably cause us a lot of headache when components will start overlapping each other. 

If this is only a draft design we can leave it for now and get rid of it later, otherwise, we definitely need to replace this.

We could use relative positioning for some popups / tooltips / dropdowns, but not for laying out the application blocks.

Reply:
Yeah, this is only a draft. This is the issue that I will be working on after finishing Display Game Result and Header PRs ([https://github.com/ZenitechSoftware/planning-chess/issues/30])